### ISSUE TITLE
Error: 'IllegalPatternException' during build

### ISSUE THREAD

**Issue by JoshStrobl** [onepub-dev/money.dart#82]
I am getting the following build error in my Flutter app when using money2 4.1.0, including when overriding fixed to 4.2.0:

```
Error: 'IllegalPatternException' is exported from both 'package:fixed/src/exceptions.dart' and 'package:money2/src/pattern_encoder.dart'.
```

In money2.dart both an export of fixed and a `export 'src/pattern_encoder.dart' show IllegalPatternException;
` , maybe the cause?

**Comment by bgetsug** [comment]
I temporarily resolved this by overriding the dependency on `fixed` in my `pubspec.yaml`:

```yaml
#...
dependency_overrides:
  #...
  fixed: 4.1.1
```

**Comment by bsutton** [comment]
I applied a fix in the fixed package to rename the exception in question which was released as 4.3.0.

running dart pub upgrade in your project that uses the money2 package should resolve the issue.
